packages/surveys/src/lib/utils.ts

```diff
@@ -69,3 +70,19 @@ export const calculateElementIdx = (survey: TSurvey, currentQustionIdx: number):
   if (possibleNextQuestions.includes("end")) elementIdx = middleIdx;
   return elementIdx;
 };
+
+export const getUpdatedTtcObj = (ttcObj: TResponseTtc, questionId: string, time: number) => {
```

Comment:
since we now have more lib functions for this I think a `lib/ttc.ts` file could make sense :-)